week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,23 +32,43 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-
+  return (
+    kinningParkHouse.currentOwner.firstName +

```

Comment:
bug.  This should return full name of the owner of the house passed in as the parameter `house`!  Currently it ALWAYS does `kinningParkHouse's owner's name, no matter what it is passed!